<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonReset">
        <property name="text">
         <string>ResetDevice</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox_frequency">
        <item>
         <property name="text">
          <string>Low frequency</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>High frequency</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_modulation">
        <item>
         <property name="text">
          <string>B-PSK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q-PSK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8-PSK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B-FSK</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B-PSK coded</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Q-PSK coded</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>B-PSK c w/PNA</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>SendDL</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_msg">
        <property name="baseSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
